README.md

```diff
@@ -273,7 +274,6 @@ To add a resource or contribute, please check out our contribution guide [here](
 - [Introduction to Git and Github - Launch School](https://launchschool.com/books/git)
 - [Git Immersion](https://gitimmersion.com/) 
 - [30SecondsOfCode Git Snippets](https://www.30secondsofcode.org/git/p/1)
-- [Git - the simple guide](https://rogerdudler.github.io/git-guide/)
```

Comment:
I'm not sure if you've accidentally removed this but please add it again. You can find it in the main repository.

Reply:
I had removed it accidentally but now I have committed my changes again. Please do the needful.